package.json

```diff
@@ -12,7 +12,8 @@
   "main": "lib/webcharts.common.js",
   "jsnext:main": "src/index.js",
   "dependencies": {
-    "d3": "^3"
+    "d3": "^3",
+    "js-xlsx": "^0.8.22"
```

Comment:
I'm thinking we should move this to [`optionalDependencies`](https://docs.npmjs.com/files/package.json#optionaldependencies)